charts/aether/values.yaml

```diff
@@ -14,7 +14,11 @@ image:
 # this secret will be volumed under `/etc/secrets`
 secretName: ""
 
-config: {}
+config:
+  providersConfig:
+    scrapingInterval: 5m
+  plugins:
+    sourceDir: /sources
```

Comment:
I dont think we should set this here, its set in the defaults: https://github.com/re-cinq/aether/blob/main/pkg/config/config.go#L42